src/lib/mockserver.js

```diff
@@ -94,25 +99,25 @@ class MockServer {
         await exec(`docker network connect ${this.network} ${this.workerContainerName}`);
     }
 
-    async stop(ignorErrors) {
+    async stop(ignoreErrors) {
         try {
-            await exec(`docker stop ${this.container}`);
+            await exec(`docker stop --time 15 ${this.container}`);
```

Comment:
Wait 15s for the container to stop, after this a SIGKILL is send to the container.

Reply:
Alternatively just do `docker rm -f <id>` right away, which is a forced kill, as we [do it for the action container](https://github.com/adobe/aio-cli-plugin-asset-compute/blob/7639af3f87417d74629ecc2f088ad289165e32cb/src/lib/actionrunner.js#L308). We don't intend to restart those containers.

Reply:
I was trying to have a clean stop, so possible open connections may have a chance to finish.
But we can also kill it, since we won't reuse it.

Changed.